pyInterface/bindings/highLevelInterface/pwaFit_py.cc

```diff
@@ -8,16 +8,36 @@
 
 namespace bp = boost::python;
 
+namespace {
+
+rpwa::fitResultPtr pwaFit_pwaFit(const rpwa::pwaLikelihood<std::complex<double> >& L,
+									const bp::dict&                                   binningMap,
+									const unsigned int                                seed = 0,
+									const std::string&                                startValFileName = "",
+									const bool                                        checkHessian = false,
+									const bool                                        saveSpace = false,
+									const bool                                        verbose = false)
```

Comment:
Please indent by one tab inside the namespace and align the arguments with spaces instead of tabs.